Automattic/woocommerce-payments
# Update wordpress/scripts and dependency-extraction-webpack-plugin

The latest version of `wordpress/scripts` and `wordpress/dependency-extraction-webpack-plugin` introduced some changes to *entrypoint.deps.json*, which became *entrypoint.asset.php*. Scripts and documentation have been updated to match those changes.

Fixes #209 

#### Testing instructions

* After switching to this branch, run `npm install` and rebuild the React client code (`npm run build` or `npm run watch`)
* You should see a new file called `index.asset.php` under the dist directory
* Everything should work as usual

-------------------
